.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           - stable
           - beta
           - nightly
-          - 1.39.0
```

Comment:
This is the MSRV.

Reply:
I can add a version detection for test builds and if too low, I will use "disable-doctest" feature (or another feature name) so you won't have build failures on versions lower than 1.40. What do you think?